filter_graph.go

```diff
@@ -154,3 +155,27 @@ func (g *FilterGraph) SendCommand(target, cmd, args string, f FilterCommandFlags
 	})
 	return
 }
+
+// https://ffmpeg.org/doxygen/7.0/structAVFilterGraph.html#a0ba5c820c760788ea5f8e40c476f9704
+func (g *FilterGraph) NbFilters() int {
+	return int(g.c.nb_filters)
+}
+
+// https://ffmpeg.org/doxygen/7.0/structAVFilterGraph.html#a1dafd3d239f7c2f5e3ac109578ef926d
+func (g *FilterGraph) Filters() (fs []*FilterContext) {
+	fcs := (*[(math.MaxInt32 - 1) / unsafe.Sizeof((*C.AVFilterContext)(nil))](*C.AVFilterContext))(unsafe.Pointer(g.c.filters))
+	for i := 0; i < g.NbFilters(); i++ {
+		fs = append(fs, newFilterContext(fcs[i]))
+	}
+	return
+}
+
+// https://ffmpeg.org/doxygen/7.0/group__lavfi.html#gadb442aca4e5a8c3ba740f6049f0a288b
+func (g *FilterGraph) Dump() string {
```

Comment:
Can you remove this function? This is already available with `FilterGraph`.`String()`